config/jobs/kubernetes/cloud-provider-openstack/release-master-presubmits.yaml

```diff
@@ -68,3 +68,20 @@ presubmits:
     annotations:
       testgrid-dashboards: provider-openstack-openstack-csi-cinder
       testgrid-tab-name: presubmit-e2e-test
+
+  - name: openstack-cloud-csi-cinder-sanity-test
+    always_run: false
+    run_if_changed: '^(pkg\/csi\/cinder\/|cmd\/cinder-csi-plugin\/)'
```

Comment:
- and `tests/sanity/cinder`?
- I noticed you didn't specify any labels, is that intentional?
- `branches` field is missing
- Does this test need to create GCP VM?
- Are you sure the kubekins-e2e image can directly run `make xxx` instead of `runner.sh make xxx`

Reply:
> Does this test need to create GCP VM?
@lingxiankong It doesnt need to interact with the cloud, so I assume not required?

Reply:
> Are you sure the kubekins-e2e image can directly run make xxx instead of runner.sh make xxx

@lingxiankong Yes I suppose. It is being used at some places directly. So I added it.
eg: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config.yaml#L38

Reply:
> and tests/sanity/cinder?

@lingxiankong yes, thanks. Will add it.

> I noticed you didn't specify any labels, is that intentional?

@lingxiankong  As I dont need here to interact with cloud, i removed them. Ill check again if required. Thanks for pointing out.

> branches field is missing

@lingxiankong  I would like to run this job on release branches as well, should I move this job to different file? A here we are adding jobs only run on master right?

Reply:
> I would like to run this job on release branches as well, should I move this job to different file? A here we are adding jobs only run on master right?

Yeah, I prefer to split CI jobs for different branches even they are the same at the moment in between (e.g. create another file `release-master-presubmits-release-1.22.yaml`), in case the job script may change in the future, we experienced the issue with OpenLab before. I see there are some repos are using the same approach.

Different branches share the same testgrid dashboards but in different tabs, e.g. `presubmit-sanity-test-release-1.22`

Reply:
@lingxiankong Ok,sounds good. Will update to master for this job and add different one for other release branches.